src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.position = 'absolute';
```

Comment:
```suggestion
```
If you look at the styles.css file you will see that class 'spider' already have position: absolute